### ISSUE TITLE
No lint issues, but gradual fails on empty compare

### ISSUE THREAD

**Issue by pboling** [skryukov/rubocop-gradual#29]
See workflow run here: https://github.com/pboling/active_security/actions/runs/9799138608/job/27058821636

```
Running RuboCop Gradual...
..................................................
Found 0 files with 0 issue(s).
Processing results...
RuboCop Gradual is complete!
RuboCop Gradual failed!

Unexpected Changes!

RuboCop Gradual lock file is outdated, to fix this message:
- Run `rubocop-gradual` locally and commit the results

`.rubocop_gradual.lock` diff:

-{
-
-}

Error: Process completed with exit code 1.
```

There is no `rubocop_gradual.lock` in the repo because there are no lint issues at all.  When I run locally I get:
```
❯ bundle exec rake rubocop_gradual
Running RuboCop Gradual...
..................................................
Found 0 files with 0 issue(s).
Processing results...
RuboCop Gradual is complete!
```
When I run the same `check` command locally I have the same problem.
```
bundle exec rake rubocop_gradual:check
Running RuboCop Gradual...
...............................................
Found 0 files with 0 issue(s).
Processing results...
RuboCop Gradual is complete!

Unexpected Changes!

RuboCop Gradual lock file is outdated, to fix this message:
- Run `rubocop-gradual` locally and commit the results

`.rubocop_gradual.lock` diff:

-{
-
-}
```

This used to work, so it must be due to a recent change somewhere.

**Comment by pboling** [comment]
@skryukov any ideas?💡

**Comment by skryukov** [comment]
Hey, @pboling, thanks for the report! 🙏 I just released `v0.3.6`, it includes the fix.
